webrtc.html

```diff
@@ -4125,13 +4125,20 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           <dt>readonly attribute DOMString? mid</dt>
 
           <dd>
-            <p>The <dfn id=
-            "dom-rtptransceiver-mid"><code>mid</code></dfn>
+            <p>The <dfn id="dom-rtptransceiver-mid"><code>mid</code></dfn>
             attribute is the <code>mid</code> negotatiated and present
             in the local and remote descriptions as defined in
-            <span data-jsep="initial-offers initial-answers">[[!JSEP]]</span>.  Before negotiation is complete,
-            the <code>mid</code> value may be null.  After rollbacks,
-            the value may change from a non-null value to null.</p>
+            <span>[[!JSEP]]</span>.
+            <!--
+            TODO add data-jsep="initial-offers initial-answers" to the span
+            -->
+            Before negotiation is complete, the <code>mid</code> value may be
+            null.  If there is no MID value in the the remote SDP, and no MID
+            value was previously assigned, a random value will be created for
+            the <code>mid</code> as described in <span
+            data-jsep="ice-candidate-format">[[!JSEP]]</span> when the remote
+            SDP is set. After rollbacks, the value may change from a non-null
```

Comment:
> On Mar 24, 2016, at 3:28 AM, Adam Bergkvist <notifications@github.com> wrote:
> 
> In webrtc.html:
> 
> >              attribute is the <code>mid</code> negotatiated and present
> >              in the local and remote descriptions as defined in
> > -            <span data-jsep="initial-offers initial-answers">[[!JSEP]]</span>.  Before negotiation is complete,
> > -            the <code>mid</code> value may be null.  After rollbacks,
> > -            the value may change from a non-null value to null.</p>
> > +            <span>[[!JSEP]]</span>.
> > +            <!--
> > +            TODO add data-jsep="initial-offers initial-answers" to the span
> > +            -->
> > +            Before negotiation is complete, the <code>mid</code> value may be
> > +            null.  If there is no MID value in the the remote SDP, and no MID
> > +            value was previously assigned, a random value will be created for
> > +            the <code>mid</code> as described in <span
> > +            data-jsep="ice-candidate-format">[[!JSEP]]</span> when the remote
> > +            SDP is set. After rollbacks, the value may change from a non-null
> 
> It would be nice to integrate the generation of mid into the 'set an RTCSessionDescription' algorithm since it would define when it time it should be done in relation to all the state updating and event dispatching.

I don't think it is really generated it then - its more when the ICE stuff happens